### PR TITLE
TRIVIAL: provide clearer error in describe group when group is inactive

### PR DESCRIPTION
**Pull request by hachikuji**

### CODE REVIEW

File: core/src/main/scala/kafka/admin/AdminClient.scala [hunk 1]
```diff
@@ -155,8 +155,11 @@ class AdminClient(val time: Time,
 
   def describeConsumerGroup(groupId: String): List[ConsumerSummary] = {
     val group = describeGroup(groupId)
+    if (group.state == "Dead")
+      throw new IllegalArgumentException(s"Group ${groupId} has no active members")
+
     if (group.protocolType != ConsumerProtocol.PROTOCOL_TYPE)
-      throw new IllegalArgumentException(s"Group ${groupId} is not a valid kafka consumer group but belongs to ${if (!group.protocolType.isEmpty) group.protocolType else "not exist"}")
+      throw new IllegalArgumentException(s"Group ${groupId} is not a valid consumer group, but belongs to ${group.protocolType}")
```
Comment (guozhangwang) [inline core/src/main/scala/kafka/admin/AdminClient.scala]: If group.protocolType is an empty string, would be better to print "but belongs to empty protocol" than "but belongs to "?

Comment (hachikuji) [inline core/src/main/scala/kafka/admin/AdminClient.scala]: This case is for non-consumer groups. The protocol type should not be empty, but maybe we can allow for that case if we use the following message:

``` scala
s"Group ${groupId} with protocol type '${group.protocolType}' is not a valid consumer group"
```

Comment (guozhangwang) [inline core/src/main/scala/kafka/admin/AdminClient.scala]: If the group does not exist, we will return empty protocol type right?

Comment (hachikuji) [inline core/src/main/scala/kafka/admin/AdminClient.scala]: That is what the 'state == "Dead"' check is for.

Comment (guozhangwang) [inline core/src/main/scala/kafka/admin/AdminClient.scala]: I missed that, sounds good.

**Comment by guozhangwang** [comment]
LGTM.

**Comment by guozhangwang** [comment]
Merged to both trunk and 0.9.0.

**Pull request closed without merging**